tests/jax/core_system_components/executor_test.py

```diff
@@ -1,5 +1,5 @@
 # python3
-# Copyright 2021 InstaDeep Ltd. All rights reserved.
+# Copyright 2022 InstaDeep Ltd. All rights reserved.
```

Comment:
I think we should keep the year 2021 (the year Mava was released) for all files. This is also what I have seen done in other repos. However, I'm of course not an expert on this, so could be wrong. But keeping the year the same would make it easier to maintain.

Reply:
Make sense to me. Updated.